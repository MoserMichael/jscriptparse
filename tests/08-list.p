    a=[1, 2, 3]
    tmp = a[0]
    a[0]=a[1]
    a[1]=a[2]
    a[2]=tmp
    println("first: {a[0]} second: {a[1]} third: {a[2]}")

