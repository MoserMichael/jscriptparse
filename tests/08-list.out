first: 2 second: 3 third: 1
